policyengine_canada/variables/gov/credits/training/training_credit.py

```diff
@@ -15,13 +15,13 @@ def formula(tax_unit, period, parameters):
         tuition = tax_unit("tuition_expenses", period)
         age = tax_unit("age", period)
         training = parameters(period).gov.credits.training
-        lower_limit = training.age_bracket_lower
-        upper_limit = training.age_bracket_upper
+        lower_limit = training.age_eligibility.min
+        upper_limit = training.age_eligibility.max
         aged = (age >= lower_limit) & (age <= upper_limit)
         existing_credits = tax_unit("existing_training_credits", period)
-        cap = training.total_cap
+        cap = training.lifetime_cap
         remaining = max_(0, cap - existing_credits)
-        threshold = training.income_threshold
+        threshold = training.amount
         credits = threshold.calc(income)
         student = tuition > 0
         eligible = min_(remaining, credits)
```

Comment:
We can't use `if` statements for vectorized computations
```suggestion
        amount_if_eligible = min_(remaining, credits)
        eligible = aged & student
        return eligible * amount_if_eligible
```